src/components/Quotes.js

```diff
@@ -2,8 +2,6 @@ import React, { useState, useEffect } from 'react';
 import './Quotes.css';
 
 const Quote = () => {
-  //   const [quote, setQuote] = useState('');
-  //   const [author, setAuthor] = useState('');
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/227177552-b59344f0-fd86-496b-b3c4-db350b459e5b.png)

**[OPTIONAL]**

- [ ] It would be great if you replace this About the Project with Math Magician because this place is for the title 🤔